src/showingpreviously/cinemas/vista_system.py

```diff
@@ -105,12 +105,29 @@ def get_showings_date(showings_data: any, chain_name: str) -> [Showing]:
     return showings
 
 
+def get_site_query(api_url: str, token: str) -> str:
+    request_headers = get_request_headers(token)
+    url = SITES_URL.format(api_url=api_url)
+    r = requests.get(url, headers=request_headers)
+    if r.status_code != 200:
+        raise CinemaArchiverException(f'Got status code {r.status_code} when fetching URL {url}')
+    try:
+        sites_data = r.json()
+    except json.JSONDecodeError:
+        raise CinemaArchiverException(f'Error decoding JSON data from URL {url}')
+    site_query = ''
+    for site in sites_data['sites']:
+        site_query += SHOWINGS_SITE_QUERY.format(site_id=site['id'])
+    return site_query[:-1]
```

Comment:
My concern here is that if there are no sites, then `site_query` will be empty, so attempting to access `site_query[:-1]` will fail. Is this something we want to account for, or is that error acceptable as a sign of a change in the API?

Reply:
I think it's an acceptable error - no cinema will have no sites (unless it's shut).